app/pages/admin-dashboard.tsx

```diff
@@ -23,8 +23,19 @@ const RightAlign = styled.div`
 `;
 
 const idpOptions = [
-  { value: ['onestopauth'], label: 'IDIR' },
-  { value: ['onestopauth-basic', 'onestopauth-business', 'onestopauth-both'], label: 'BCeID' },
+  { value: ['onestopauth', 'idir', 'azureidir'], label: 'IDIR' },
```

Comment:
@thegentlemanphysicist idir and azureidir are parent realms and only standard realm has all the gold integrations. Any reason why they were included?

Reply:
yes, I just about to say that. onestopauth is a standard realm in silver, while idir and azureidir are parent realm in gold. Scratching my head as I don't understand why there would be any client data in parent realm:)